base/libc.jl

```diff
@@ -77,6 +77,18 @@ flush_cstdio() = ccall(:jl_flush_cstdio, Void, ())
 @unix_only systemsleep(s::Real) = ccall(:usleep, Int32, (UInt32,), round(UInt32,s*1e6))
 @windows_only systemsleep(s::Real) = (ccall(:Sleep, stdcall, Void, (UInt32,), round(UInt32,s*1e3)); return Int32(0))
 
+type TimevalStruct
+   sec::Clong   # time_t
```

Comment:
I was going to say we shpmould probably prefix this if we're going to be exporting it from libjulia